scipy/signal/ltisys.py

```diff
@@ -588,41 +382,245 @@ def freqresp(self, w=None, n=10000):
         """
         return freqresp(self, w=w, n=n)
 
+    def to_discrete(self, dt, method='zoh', alpha=None):
+        """Return a discretized version of the current system.
+
+        Parameters: See `cont2discrete` for details.
+
+        Returns
+        -------
+        sys: instance of `dlti`
+        """
+        raise NotImplementedError('to_discrete is not implemented for this '
+                                  'system class.')
+
+
+class dlti(LinearTimeInvariant):
+    """
+    Discrete-time linear time invariant system base class.
+
+    Parameters
+    ----------
+    *system: arguments
+        The `dlti` class can be instantiated with either 3, 4 or 5 arguments.
```

Comment:
There's some asymmetry with ``lti`` here - ``lti`` docstring says "2, 3 or 4 arguments". Implementation and length checks in ``__new__`` look the same